src/map/homunculus.cpp

```diff
@@ -573,6 +559,7 @@ int hom_levelup(struct homun_data *hd)
 	// Needed to update skill list for mutated homunculus so unlocked skills will appear when the needed level is reached.
 	status_calc_homunculus(hd,SCO_NONE);
 	clif_hominfo(hd->master,hd,0);
+	clif_homunculus_updatestatus(*hd->master, SP_BASEEXP);
```

Comment:
Why do we need to update here?
The EXP are part of clif_hominfo.

Reply:
Official server sends it after hominfo here